kuma/settings/common.py

```diff
@@ -954,8 +954,9 @@ def pipeline_one_scss(slug, **kwargs):
         'source_filenames': (
             'js/payments-handler.js',
             'js/payments-faq.js',
+            'js/payments-confirmation.js',
```

Comment:
Should this be part of the always-running JS, or should it be in a separate JS file that is only loaded in the ``payments/thank_you.html`` template?

Reply:
I think only loading this in the confirmation pages makes more sense, I'll quickly update this PR so it does that!